packages/sui-studio/bin/sui-studio-dev.js

```diff
@@ -45,21 +45,26 @@ if (!category || !component) {
 
 const componentPath = path.join(PWD, 'components', category, component)
 const legacyTestPath = path.join(PWD, 'test', category, component)
+const jestPath = path.join(componentPath, '__tests__')
 
 const testPath = fs.existsSync(legacyTestPath) ? legacyTestPath : path.join(componentPath, 'test')
+const isJestTest = fs.existsSync(jestPath)
 
 const {cache, ...others} = config
 
 const studioDevConfig = {
   ...others,
   context: path.join(__dirname, '..', 'workbench', 'src'),
-  plugins: [...config.plugins, new webpack.DefinePlugin({__COMPONENT_ID__: JSON.stringify(componentID)})],
+  plugins: [
+    ...config.plugins,
+    new webpack.DefinePlugin({__COMPONENT_ID__: JSON.stringify(componentID), __DISABLE_TESTS__: isJestTest})
+  ],
   resolve: {
     ...config.resolve,
     alias: {
       ...config.resolve.alias,
       component: path.join(componentPath, 'src'),
-      test: testPath,
+      test: isJestTest ? jestPath : testPath,
```

Comment:
I think that now this is not needed

Reply:
Totally agree.

Reply:
@andresz1 I only keep the testPath, isn't it?

`test: testPath`